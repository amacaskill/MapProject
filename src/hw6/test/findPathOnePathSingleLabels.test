# One path, where the path only includes single labels between two nodes 

CreateGraph g 

AddNode g char1
AddNode g char2
AddNode g char3
AddNode g char4

AddEdge g char1 char2 label1
AddEdge g char2 char3 label2
AddEdge g char3 char4 label3 

FindPath g char1 char4