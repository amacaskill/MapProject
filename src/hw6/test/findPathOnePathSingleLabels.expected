# One path, where the path only includes single labels between two nodes 

created graph g 

added node char1 to g
added node char2 to g
added node char3 to g
added node char4 to g

added edge label1 from char1 to char2 in g
added edge label2 from char2 to char3 in g
added edge label3 from char3 to char4 in g

path from char1 to char4:
char1 to char2 via label1
char2 to char3 via label2
char3 to char4 via label3